hatch/src/cli/commands/new.rs

```diff
@@ -69,7 +75,7 @@ impl<'command> Command<'command> for New {
 
       .arg(Arg::with_name(PROJECT_NAMES)
            .help("Name of project")
-           .takes_value(true).max_values(1)
+           .takes_value(true)
```

Comment:
why does this now take more than one?

Reply:
It doesnt, we just dont need to specify the number of values it takes now because we never specify that it takes more than 1